hibernate-core/src/main/java/org/hibernate/type/descriptor/java/LocalDateTimeJavaType.java

```diff
@@ -99,12 +99,12 @@ public <X> X unwrap(LocalDateTime value, Class<X> type, WrapperOptions options)
 
 		if ( java.sql.Date.class.isAssignableFrom( type ) ) {
 			Instant instant = value.atZone( ZoneId.systemDefault() ).toInstant();
-			return (X) java.sql.Date.from( instant );
+			return (X) new java.sql.Date( instant.toEpochMilli() );
 		}
 
 		if ( java.sql.Time.class.isAssignableFrom( type ) ) {
 			Instant instant = value.atZone( ZoneId.systemDefault() ).toInstant();
-			return (X) java.sql.Time.from( instant );
+			return (X) new java.sql.Time( instant.toEpochMilli() );
```

Comment:
Maybe we should also `% 86_400_000` here, like we did for [JdbcTimestampJavaType.java](https://github.com/hibernate/hibernate-orm/pull/8833/files#diff-12951dffbbe474145b0aae135041257097fc298bf3fe918a9c72efe34ca34cdc), as recommended in `java.sql.Time` [javadoc](https://docs.oracle.com/javase/8/docs/api/java/sql/Time.html).

Reply:
I can not find that in `ȷava.sql.Time` javadoc, but try to run this:

```
		TimeZone.setDefault(TimeZone.getTimeZone( "Africa/Monrovia" ));
		System.out.println(new java.sql.Time(0).toLocalTime());
```

I simply love this zone for testing, time offset on January 1st 1970 was 44 minutes and 30 seconds :-)